### PR TITLE
Add js example for list query usage

### PR DESCRIPTION
**Pull request by thakkerurvish**
TSIA

Fixes: TF-3907

### CODE REVIEW

**Comment by thakkerurvish** [comment]
I'd accidentally committed my AgentQL API key:(
As a mitigation step, I deleted all my production keys including the one which was committed. So, no longer that would be valid. Sorry about that!

File: javascript-sdk/examples/list-query-usage/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Example script: querying a list of items with AgentQL
+
+This example demonstrates how to query a list of items on the page.
+
+## Run the script
+
+- [Install AgentQL SDK](https://agentql-docs-opvm8e9cd-tiny-fish.vercel.app/javascript-sdk/installation)
+- Save this javascript file locally as **list-query-usage.js**
```
Comment (rachelnabors) [inline javascript-sdk/examples/list-query-usage/README.md:8]:
```suggestion
- Save this JavaScript file locally as **list-query-usage.js**
```

**Review by rachelnabors — APPROVED** [review]

File: javascript-sdk/examples/list-query-usage/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Example script: querying a list of items with AgentQL
+
+This example demonstrates how to query a list of items on the page.
+
+## Run the script
+
+- [Install AgentQL SDK](https://agentql-docs-opvm8e9cd-tiny-fish.vercel.app/javascript-sdk/installation)
```
Comment (rachelnabors) [inline javascript-sdk/examples/list-query-usage/README.md:7]:
```suggestion
- [Install AgentQL SDK](https://docs.agentql.com/javascript-sdk/installation)
```

File: javascript-sdk/examples/list-query-usage/list-query-usage.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+const path = require('path');
+const fs = require('fs');
+
+(async () => {
+  // Set the AgentQL API key via the `configure` method.
+  configure({ apiKey: process.env.AGENTQL_API_KEY });
```
Comment (rachelnabors) [inline javascript-sdk/examples/list-query-usage/list-query-usage.js:8]: I recommend moving the keys to the installation docs, not the example files

**Review by rachelnabors — CHANGES_REQUESTED** [review]

File: javascript-sdk/examples/list-query-usage/README.md (hunk 1 above)
Comment (thakkerurvish) [reply javascript-sdk/examples/list-query-usage/README.md]: Updated it!

File: javascript-sdk/examples/list-query-usage/README.md (hunk 2 above)
Comment (thakkerurvish) [reply javascript-sdk/examples/list-query-usage/README.md]: Updated it!

File: javascript-sdk/examples/list-query-usage/list-query-usage.js (hunk 3 above)
Comment (thakkerurvish) [reply javascript-sdk/examples/list-query-usage/list-query-usage.js]: Removed it from the example.

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**